### PR TITLE
Update attribute key of rocketmq's message tag

### LINKED ISSUE #6676: Update attribute key of rocketmq's message tag
Since the [semantic convetions of rocketmq](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#apache-rocketmq) has been merged into [opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification), the attribute key of rocketmq's message tag should be updated.

### PR DESCRIPTION
**Pull request by aaron-ai**
Fixes #6676

### CODE REVIEW

File: instrumentation/rocketmq-client-4.8/library/src/main/java/io/opentelemetry/instrumentation/rocketmq/RocketMqConsumerExperimentalAttributeExtractor.java [hunk 1]
```diff
@@ -17,8 +17,8 @@
     implements AttributesExtractor<MessageExt, Void> {
   INSTANCE;
 
-  private static final AttributeKey<String> MESSAGING_ROCKETMQ_TAGS =
-      AttributeKey.stringKey("messaging.rocketmq.tags");
+  private static final AttributeKey<String> MESSAGING_ROCKETMQ_MESSAGE_TAG =
+      AttributeKey.stringKey("messaging.rocketmq.message_tag");
```
Comment (trask) [inline instrumentation/rocketmq-client-4.8/library/src/main/java/io/opentelemetry/instrumentation/rocketmq/RocketMqConsumerExperimentalAttributeExtractor.java:21]: it looks like there's a constant for this in the `SemanticAttributes` class that we could instead

**Review by trask — APPROVED** [review]
thx @aaron-ai! left one comment above

**Comment by aaron-ai** [comment]
> 

@trask got it, `messaging.rocketmq.tags` has been replaced by the constant in `SemanticAttributes `.

**Comment by trask** [comment]
thx @aaron-ai!

**Pull request merged**